### PR TITLE
Do you have tickets??

### PR DESCRIPTION
**Pull request by msft-jasonparker**

### CODE REVIEW

File: _slides/09-msft-jasonparker.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+# Welcome to the Gun Show
```
Comment (AConant) [inline _slides/09-msft-jasonparker.md:6]: got a permit for that?

**Review by AConant — APPROVED** [review]
allowed by law...for now...

**Comment by AConant** [comment]
you may proceed...but I'm watching you!

File: _slides/09-msft-jasonparker.md (hunk 1 above)
Comment (msft-jasonparker) [reply _slides/09-msft-jasonparker.md:6]: It's a conceal carry permit

**Pull request merged**